cms/migrations/0019_auto_20161005_2008.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-10-05 20:08
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0028_merge'),
+        ('cms', '0018_update_programfaculty'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FaqsPage',
+            fields=[
+                ('page_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailcore.Page')),
+            ],
+            options={
+                'abstract': False,
+            },
+            bases=('wagtailcore.page',),
+        ),
+        migrations.RemoveField(
+            model_name='frequentlyaskedquestion',
+            name='program_page',
+        ),
+        migrations.AddField(
+            model_name='frequentlyaskedquestion',
+            name='faqs_page',
+            field=modelcluster.fields.ParentalKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='faqs', to='cms.FaqsPage'),
+        ),
+    ]
```

Comment:
This is @giocalitri :
this migration cannot be reverted and should not be merged as is.

This file should be renamed to something meaningful